test/test_example.py

```diff
@@ -65,3 +65,7 @@ def test_flaky_thing_that_fails_then_succeeds(self):
         self._threshold += 1
         if self._threshold < 1:
             raise Exception("Threshold is not high enough.")
+
+
+def test_function():
```

Comment:
In this example test module should there be more 'example' of this test function? Like tagging it flaky, etc?